mabl snippets/GetDateComponents.js

```diff
@@ -34,33 +34,43 @@ This then can be combined to create dates formatted however you need:
   "{{@date.yearNumeric}}-{{@date.monthTwoDigit}}-{{@date.dayTwoDigit}}" => "2021-02-07"
 */
 
+// The locale used to format the dates
+let locale = "en-us";
```

Comment:
Nit: should this be `en-US`?

Reply:
It seems to work, but I think you are right.